docs/guides/doingatransformation.md

```diff
@@ -1,20 +1,30 @@
 # Doing a Transformation
 
-The Transformation object in TriggerMesh defines a set of operations that are
+The `Transformation` object in TriggerMesh defines a set of operations that are
 sequentially applied to incoming CloudEvents. In this guide, we will create a
-simple flow with an event producer and transformation to see the declarative syntax
-that is used for modifying events.
+simple flow (a.k.a Bridge) with an event producer and a transformation to see the declarative syntax that is used for modifying events.
 
-## Prerequisites
+!!! Info "Prerequisites"
+    You need a working TriggerMesh platform installation. See the [installation steps](installation.md). You can verify that the API is available with the following command:
 
-1. K8s cluster and configured kubectl
-1. [Knative Serving and Eventing Operators](https://knative.dev/docs/admin/install/knative-with-operators/)
-1. [TriggerMesh Bumblebee](https://github.com/triggermesh/bumblebee)
+    ```console
+       $ kubectl get crd|grep transformation
+       transformations.flow.triggermesh.io                     2021-10-06T09:01:40Z
+    ```
```

Comment:
grep is totally unnecessary here.
```suggestion
    ```console
    $ kubectl get crd transformations.flow.triggermesh.io
    NAME                                  CREATED AT
    transformations.flow.triggermesh.io   2021-10-06T09:01:40Z
    ```
```